app/views/profiles/edit.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="row">
+  <h1>Your Profile</h1>
+  <div class="col-xs-12">
+
+
+  <%= form_for @profile do |f| %>
+    <fieldset>
+      <%= f.label "I drink" %>
+      <%= f.radio_button 'drinker', true %>
+      <%= f.label "I don't drink" %>
+      <%= f.radio_button 'drinker', false %>
```

Comment:
I think that these labels wouldn't be bound to the radio buttons quite correctly- I think you'd have one label for the radio button group?